h/views/main.py

```diff
@@ -57,14 +57,24 @@ def robots(context, request):
 
 @view_config(route_name='stream')
 def stream(context, request):
-    atom = request.route_url('stream_atom')
-    rss = request.route_url('stream_rss')
-    return render_app(request, {
-        'link_tags': [
-            {'rel': 'alternate', 'href': atom, 'type': 'application/atom+xml'},
-            {'rel': 'alternate', 'href': rss, 'type': 'application/rss+xml'},
-        ]
-    })
+    if request.feature('search_page'):
+        q = request.params['q'].split(':')
+        if q[0] == 'tag':
+            tag = q[1]
+            if tag.find(' ') > -1:
+                tag = "'" + tag + "'"
+            query = {'q': 'tag:{}'.format(tag)}
```

Comment:
Nice making sure that the different behaviour with whitespace in the stream and activity pages is cared for!